corral/cmd/_test_cmd_update.pony

```diff
@@ -37,6 +39,44 @@ class iso _TestEmptyDeps is UnitTest
     h.long_test(2_000_000_000)
 
 
+class iso _TestRegression120 is UnitTest
+  fun name(): String =>
+    "cmd/update/" + __loc.type_name()
+
+  fun apply(h: TestHelper) ? =>
+    """
+    Issue #120 identified a problem with transitive dependencies that resulted
+    in too many operations being performaned across the loading of all
+    dependencies.
+
+    The test as currently constituted, consists of a bundles with 2
+    dependencies. One of those has 2 more in a transitive fashion, that should
+    result in 4 syncs and corresponding actions happening. However, due to a
+    bug in _Updater, it currently does 11.
+
+    With a real VCS like Git, the number that results from it is variable
+    based on timing. This test exists to prove that issue #120 is fixed and
+    to prevent a similar bug from being introduced in the future.
+    """
+    // given
+    let auth = h.env.root as AmbientAuth
+    let log = Log(LvlNone, h.env.err, SimpleLogFormatter)
+    let fp: FilePath = _TestData.file_path_from(h, "regression-120/bundle-entrypoint")?
+    let repo_cache = _TestRepoCache(auth)?
+    let ctx = Context(h.env, log, log, false, repo_cache)
+    let project = Project(auth, log, fp)
+    let bundle = Bundle.load(fp, log)?
+    let recorder = _OpsRecorder(h, 4, 4, 4)
+    let vcs_builder: VCSBuilder = _TestCmdUpdateVCSBuilder(recorder)
+
+    // when
```

Comment:
Is this comment intended to be here?

Reply:
Yes...

There's also a "given" above before the setup so Given this setup, when Blah...

from a given, when, then style.

If you think it is confusing, I can remove.

Reply:
I think it can stay, since I don't think it would hinder anyone's understanding of the test